apps/dcos_dns/src/dcos_dns_mesos.erl

```diff
@@ -140,7 +140,9 @@ task_records(_TaskId, Task) ->
 -spec(task_agentip(task()) -> [dns:dns_rr()]).
 task_agentip(#{name := Name, framework := Fwrk, agent_ip := AgentIP}) ->
     DName = format_name([Name, Fwrk, <<"agentip">>], ?DCOS_DOMAIN),
-    dns_records(DName, [AgentIP]).
+    dns_records(DName, [AgentIP]);
+task_agentip(_Task) ->
+    [].
```

Comment:
Shall we log these tasks as WARNING, so that it is easier to reason missing DNS?